scanpipe/pipes/scancode.py

```diff
@@ -350,14 +366,7 @@ def scan_resources(
                     "Please ensure that there is at least 2 GB of available memory per "
                     "CPU core for successful execution."
                 )
-
-                resource.project.add_error(
-                    exception=broken_pool_error,
-                    model="scan_resources",
-                    description=message,
-                    object_instance=resource,
-                )
-                continue
+                raise broken_pool_error from InsufficientResourcesError(message)
```

Comment:
Why are we reverting to the old behavior?

Reply:
This was problematic at scale with multiprocessing and creating a lot more errors other than the resource which caused the memory spike, which was not apparent at first from testing smaller codebases.